### PR TITLE
Add config option to control SSL validation

### PR DESCRIPTION
**Pull request by tsipinakis**
Currently newsbeuter performs no validation on SSL certificates. Until a better solution is implemented (per feed SSL control) a global setting is better than doing no validation at all.

### CODE REVIEW

File: doc/example-config [hunk 1]
```diff
@@ -916,6 +916,16 @@
 #
 # prepopulate-query-feeds yes
 
+###  ssl-verify
```
Comment (Minoru) [inline doc/example-config]: There should be four hash symbols, not three ;)

**Comment by Minoru** [comment]
I apologize for the delay.

JFTR: this PR makes SSL verification enabled by default, which is backwards incompatible. But I view this more like "setting things right" rather than "breaking what we already have", so I'll merge this as it is once that little nitpick I found gets resolved :)

File: doc/example-config (hunk 1 above)
Comment (tsipinakis) [inline doc/example-config]: Fixed

**Comment by Minoru** [comment]
Merged. Thank you!

**Pull request merged**